k8s/agones/Makefile

```diff
@@ -0,0 +1,112 @@
+include ../app.Makefile
+include ../crd.Makefile
+include ../gcloud.Makefile
+include ../var.Makefile
+
+TAG ?= 0.8.1
+DEMO_RELEASE_TAG ?= release-0.8.0
```

Comment:
Shouldn't this be `release-0.8.1`?

Reply:
Not necessarily. The example section of Agones hasn't changed in a while, so using 0.8.0 is more than enough. And Agones 0.8.1 hasn't been officially released since we are waiting to make sure this part of the deployment doesn't need any more hotfixes.

Reply:
Does not apply anymore.

Reply:
Does not apply anymore.

Reply:
Done